src/ray/raylet/worker_pool.cc

```diff
@@ -427,11 +427,12 @@ void WorkerPool::MarkPortAsFree(int port) {
 }
 
 void WorkerPool::HandleJobStarted(const JobID &job_id, const rpc::JobConfig &job_config) {
-  unfinished_jobs_[job_id] = job_config;
+  all_jobs_[job_id] = job_config;
 }
 
 void WorkerPool::HandleJobFinished(const JobID &job_id) {
-  unfinished_jobs_.erase(job_id);
+  // Currently we don't erase the job from `all_jobs_` , as a workaround for
+  // https://github.com/ray-project/ray/issues/11437.
```

Comment:
Can we just keep `unfinished_jobs_.erase(job_id);` commented in?

Reply:
done

Reply:
Maybe change to `// all_jobs_.erase(job_id);`?